<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font-family: Poppins;
font-size:20px;
font-weight: 500;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	height: 50px;
	border-radius: 25px;
	border: 1px solid #00007f;
	background-color: white;
	color: #00007f;
}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="monitoringBtn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Monitoreo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calibrationBtn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Calibraci√≥n</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dataBtn">
       <property name="text">
        <string>Datos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bluetoothBtn">
       <property name="text">
        <string>Bluetooth</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
